src/components/now.rs

```diff
@@ -10,35 +10,110 @@ use crate::{iso::IsoDateTime, TemporalUnwrap};
 use super::{
     calendar::Calendar,
     timezone::{TimeZone, TzProvider},
-    EpochNanoseconds, Instant, PlainDateTime,
+    EpochNanoseconds, Instant, PlainDate, PlainDateTime, PlainTime, ZonedDateTime,
 };
 
+#[cfg(feature = "experimental")]
+use crate::{components::timezone::TZ_PROVIDER, TemporalError};
+#[cfg(feature = "experimental")]
+use std::ops::Deref;
```

Comment:
Hmm, I think if using `deref` needs to be gated, it would be better to use the `*` syntax.

Reply:
Yeah, I wasn't the biggest fan of using it at the time. I'm fine removing it.